lib/iris/tests/results/imagerepo.json

```diff
@@ -1,725 +1,726 @@
 {
     "example_tests.test_COP_1d_plot.TestCOP1DPlot.test_COP_1d_plot.0": [
-        "https://scitools.github.io/test-iris-imagehash/images/5dff1a996c06b47193eecc52275b936d58b6239ba94c133643897c6c2333f086.png"
-    ],
+        "https://scitools.github.io/test-iris-imagehash/images/v4/baff589936602d8ec977334ae4dac9b61a6dc4d99532c86cc2913e36c4cc0f61.png"
```

Comment:
This is a bit awkward, because technically I should double check that the old hash matches the new (and that a change hasn't been introduced).

I'm not going to do that, and rely on the fact that the tests passed before this change, and that they will pass after this change + the fact that no changes have occurred that change the behaviour of Iris' plotting...

Reply:
Yeah, it's really pretty difficult for a reviewer to review the hash changes, but I've tried to address that some what with this https://github.com/SciTools/iris/pull/2989#issuecomment-377736755